transformers/asteroids

```diff
@@ -0,0 +1,23 @@
+const { getAverageValue } = require('../helpers/math');
```

Comment:
you forgot about the file extension
it works?

Reply:
Yes. It works. But I really forgot to add .js extension some files. Now I've fixed it.